modules/ztp-sno-du-subscribing-to-the-operators-needed-for-platform-configuration.adoc

```diff
@@ -12,38 +12,45 @@
 * Logging Operator
 * PTP Operator
 * SR-IOV Network Operator
+* SRIOV-FEC Operator
```

Comment:
This one is optional

Reply:
Since it's optional, I will remove SRIOV-FEC altogether.